sklearn/gaussian_process/kernels.py

```diff
@@ -685,10 +685,22 @@ def requires_vector_input(self):
 
 
 class Sum(KernelOperator):
-    """Sum-kernel k1 + k2 of two kernels k1 and k2.
+    """The `Sum` kernel takes two kernels :math:`k_1` and :math:`k_2`
+    and combines them via
 
-    The resulting kernel is defined as
-    k_sum(X, Y) = k1(X, Y) + k2(X, Y)
+    .. math::
+        k_{sum}(X, Y) = k_1(X, Y) + k_2(X, Y)
+
+    Using the Sum-kernel is equivalent to using the + operator::
+
+        RBF() + RBF()
+
+    is the same as::
+
+        Sum(RBF(), RBF())
```

Comment:
OK I was reading this wrong and I agree this is useful. I'd just suggest to shorten it in one line, i.e.:

> Note that the `__add__` magic method is overridden, so `Sum(RBF(), RBF())` is equivalent to using the `+` operator with `RBF() + RBF()`.

here and in the other Kernels